pkg/app/server/grpcapi/grpcapi.go

```diff
@@ -191,6 +191,22 @@ func getEncriptionKey(se *model.Piped_SecretEncryption) ([]byte, error) {
 	}
 }
 
+func gRPCErrorForEntityOperation(err error, op string, a ...interface{}) error {
+	if err == nil {
+		return nil
+	}
+	switch err {
+	case datastore.ErrNotFound:
```

Comment:
nit: it's a little bit weird to evaluate err twice, but it's up to you.
```suggestion
	switch err {
	case nil:
	        return nil
	case datastore.ErrNotFound:
```